MaxMind.MinFraud.UnitTest/MaxMind.MinFraud.UnitTest.csproj

```diff
@@ -3,11 +3,10 @@
   <PropertyGroup>
     <Description>Test project for minFraud API</Description>
     <VersionPrefix>0.6.0</VersionPrefix>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net452;net46;net5.0;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net461;net5.0;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">net5.0;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
     <AssemblyName>MaxMind.MinFraud.UnitTest</AssemblyName>
     <AssemblyOriginatorKeyFile>../MaxMind.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
```

Comment:
We are no longer using the friend assembly feature so we don't need to sign the unit tests.